runeq/resources/common.py

```diff
@@ -62,10 +62,13 @@ def __repr__(self):
         Format the item representation.
 
         """
-        return (
-            f'{self.__class__.__name__}(\n\tid={self._id},\n\t'
-            f'attributes={self._attributes}\n)'
-        )
+        if hasattr(self, "name"):
+            return (
+                f'{self.__class__.__name__}'
+                f'(id="{self.id}", name="{self.name}")'
```

Comment:
nitpick: I would add in some line breaks so that the super long id doesn't push the name out of the way.

Reply:
I *think* it looks okay when I run it locally... I think the worst item is `Device`, because it overrides the method to include the device ID, name, _and_ patient ID. With one of the longer-named devices, the string is still coming in at <100 characters, which seems fine for a single line. I have a preference for the single-line format, because it makes it compact to print out items in a loop.